bicycleparameters/cycle_app.py

```diff
@@ -399,5 +396,4 @@ def input_triggers_spinner2(value):
 
 
 if __name__ == '__main__':                         # omit the `dev_tools_ui` parameter to display debug info 
-    app.run_server(debug=True, dev_tools_ui=False) # in the browser rather than in the terminal
-    
+    app.run_server(debug=True, dev_tools_ui=True) # in the browser rather than in the terminal
```

Comment:
I prefer to leave the dev tools off in the main code. I find it a bit annoying (it's some newer feature of dash)